pkg/config/config.go

```diff
@@ -20,6 +21,8 @@ const (
 type OdoSettings struct {
 	// Controls if an update notification is shown or not
 	UpdateNotification *bool `json:"updatenotification,omitempty"`
+	// Holds the prefix part of generated random application name
+	Prefix *string `json:"prefix,omitempty"`
```

Comment:
Would it be a good idea to name this App-Prefix or something like that? I think it would be useful if we later decide to have prefixes for components too, WDYT?

Reply:
Just a thought to be future proof. In future if we decide to generate component names randomly, in that case having same Prefix for both app and component might be confusing as they start with the same prefix like `blah-peru` and `blah-something` for both app name and component name so might be better to separate them.

Reply:
ok so generally, I feel the prefix is a way of identifying who created the resource so immaterial, of the resource type, the prefix is expected to be same in my opinion for a person/group of persons creating the resource

@kadel or @jorgemoralespou can give us a better idea of the use-case for prefix

Reply:
I would also use a little bit more specific name than just Prefix. It is too generic.
Ideally, it should be a little bit descriptive and give the user a hint what this configuration means.

Reply:
Would AutoGenNamePrefix be good enough?